gomod2nix.toml

```diff
@@ -0,0 +1,27 @@
+schema = 3
+
+[mod]
+  [mod."github.com/creack/pty"]
+    version = "v1.1.21"
```

Comment:
How will this dependency data be kept in sync with the *go.mod* file? If someone submits a change that updates the version of any of these packages (which I have done more than once) do they also have to update this file or is file somehow automatically kept in sync with the *go.mod* file for the project?

Reply:
This has to be updated together with the command 
```nix develop --command gomod2nix```.

If adding flake is OK, I'll try to add workflow that generates automatic PR when dependancy changes.